frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -116,7 +123,18 @@ const PVCTableRow = ({ obj, index, key, style }) => {
       <TableData className={tableColumnClasses[4]}>
         {_.get(obj, 'status.capacity.storage', '-')}
       </TableData>
-      <TableData className={tableColumnClasses[5]}>
+      <TableData className={classNames(tableColumnClasses[5])}>
+        {obj?.spec?.storageClassName ? (
+          <ResourceLink
+            kind="StorageClass"
+            name={obj?.spec?.storageClassName}
```

Comment:
You should be able to just specify these as ```obj.spec.storageClassName``` since you've already run the check above.

Reply:
However, this is a backport.  If we want the change, we should prob make it in 4.5 and backport again (else the change doesn't persist into future versions).

Reply:
I'd drop this commit here, since its a backport, and just make a new PR for 4.5 only.  We can take the change moving forward.

Reply:
/hold

move the second commit into a separate PR so that the backport is clean, thanks!

Reply:
> /hold
> 
> move the second commit into a separate PR so that the backport is clean, thanks!

@benjaminapetersen Done! Please review.